### PR TITLE
Wcx date theme

### PR DESCRIPTION
**Pull request by cuicuiwang**

### CODE REVIEW

File: src/widgets/date-picker/themeConfig/themeConfig.js [hunk 1]
```diff
@@ -1,16 +1,173 @@
...
+  const { themeConfig } = themeProps;
+  const defaultNormal = {
+    normal: {
+      width: '100%',
+      height: 32,
+      border: {
+        top: { width: borderSize, color: borderColor, style: 'solid' },
+        right: { width: borderSize, color: borderColor, style: 'solid' },
+        bottom: { width: borderSize, color: borderColor, style: 'solid' },
+        left: { width: borderSize, color: borderColor, style: 'solid' },
```
Comment (zenjava) [inline src/widgets/date-picker/themeConfig/themeConfig.js]: 重复代码

File: src/widgets/date-picker/themeConfig/themeConfig.js [hunk 2]
```diff
@@ -1,16 +1,173 @@
...
+      color: borderDisableColor,
+    },
+    borderRadius,
+  };
+  const hoverTheme = deepMerge(hover, deafultHoverBorderColor);
+  const disabledTheme = deepMerge(defaultDisabled, disabled);
+  themeProps.themeConfig.normal = normal;
+  themeProps.themeConfig.hover = hoverTheme;
+  themeProps.themeConfig.disabled = disabledTheme;
+  return themeProps;
```
Comment (zenjava) [inline src/widgets/date-picker/themeConfig/themeConfig.js:78]: const {themeConfig} = themeProps;
 themeConfig.normal = normal;
themeConfig.hover = hoverTheme;
themeConfig.disabled = disabledTheme;

File: src/widgets/date-picker/themeConfig/themeConfig.js [hunk 3]
```diff
@@ -1,16 +1,166 @@
...
+  const defaultNormal = {
+    color: '#666',
+  };
+  const defaultHover = {
+    background: { color: hoverColor },
+    color: '#fff',
+  };
+  const defaultActive = {
+    background: { color: normalColor },
+    color: '#fff',
```
Comment (SleepyAries) [inline src/widgets/date-picker/themeConfig/themeConfig.js:124]: darkGreyColor: '#666666', //正文色  defaultColor: '#ffffff', //默认色 lightGreyColor: '#cccccc', //水印色

File: src/widgets/date-picker/themeConfig/themeConfig.js [hunk 4]
```diff
@@ -1,16 +1,166 @@
...
+  };
+}
+export function getSecondWeekDateTheme(props) {
+  const { getPartOfThemeProps } = props;
+  const themeProps = getPartOfThemeProps('SecondWeekDate');
+  const {
+    themeConfig: { normal = {}, hover = {} },
+  } = themeProps;
+  const defaultNormal = {
+    color: '#333',
```
Comment (SleepyAries) [inline src/widgets/date-picker/themeConfig/themeConfig.js:153]: blackColor: '#333333', //标题色

File: src/widgets/date-picker/styled/styled.js [hunk 5]
```diff
@@ -586,27 +539,27 @@ const getDateChildStyle = props => {
       todayStyle = `
       &:nth-child(${todayInd})>i{
         border:1px solid transparent;
-        border-radius:50%;       
+        border-radius:50%;
```
Comment (SleepyAries) [inline src/widgets/date-picker/styled/styled.js:542]: circleBorderRadius: '50%', //圆角-特殊

File: src/widgets/date-picker/styled/styled.js [hunk 6]
```diff
@@ -586,27 +539,27 @@ const getDateChildStyle = props => {
...
       }
   `;
     }
     chooseWeeks = `
     background:${backG};
-    
+
       &>i{
         color:#fff;
         border-radius:50%;
```
Comment (SleepyAries) [inline src/widgets/date-picker/styled/styled.js:551]: circleBorderRadius: '50%', //圆角-特殊

File: src/widgets/date-picker/styled/styled.js [hunk 7]
```diff
@@ -551,30 +499,35 @@ const getDateChildStyle = props => {
...
     }`;
   }, '');
   const todayInd = noToday ? '' : selectToday ? todayIndex : '';
   let todayStyle = `
       &:nth-child(${todayInd})>i{
-        border:1px solid ${normalColor};        
+        border:1px solid ${normalColor};
         color:#666;
-        background:#fff;      
+        background:#fff;
```
Comment (SleepyAries) [inline src/widgets/date-picker/styled/styled.js:522]: defaultColor: '#ffffff', //默认色  borderSize: 1, //边框大小

File: src/widgets/date-picker/styled/styled.js [hunk 8]
```diff
@@ -551,30 +499,35 @@ const getDateChildStyle = props => {
...
         border-radius:50%;
```
Comment (SleepyAries) [inline src/widgets/date-picker/styled/styled.js:523]: circleBorderRadius: '50%', //圆角-特殊

File: src/widgets/date-picker/styled/styled.js [hunk 9]
```diff
@@ -551,30 +499,35 @@ const getDateChildStyle = props => {
...
+    color,
+  } = activeTheme;
   const chooseStyle = arrChoseDayIndex.reduce((p, n) => {
     return `${p}
     &:nth-child(${n})>i{
-      background:${normalColor};
-      color:#fff;
+      background:${bgColor};
+      color:${color};
       border-radius:50%;
```
Comment (SleepyAries) [inline src/widgets/date-picker/styled/styled.js:514]: circleBorderRadius: '50%', //圆角-特殊

File: src/widgets/date-picker/styled/styled.js [hunk 10]
```diff
@@ -335,15 +373,26 @@ export const DateChildInner = CSSComponent({
     cursor: pointer;
     font-size: ${em(14)};
     &:hover {
-      background: ${hoverColor};
-      color: #fff;
+      ${props => getHoverStyle(props)};
       border-radius: 50%;
```
Comment (SleepyAries) [inline src/widgets/date-picker/styled/styled.js:377]: circleBorderRadius: '50%', //圆角-特殊

File: src/widgets/date-picker/panel/RangeInput.js [hunk 11]
```diff
@@ -120,63 +123,65 @@ class RangeInput extends Component<TypeProps, TypeState> {
...
             },
+            InputSuffix: {
+              disabled: {
+                color: '#ddd',
+              },
+              ...inputSuffixProps.themeConfig,
+            },
             ClearButton: {
+              disabled: {
+                color: '#ddd',
```
Comment (SleepyAries) [inline src/widgets/date-picker/panel/RangeInput.js]: disableColor: '#f2f2f2', //禁止色

**Pull request closed without merging**